vendor/github.com/Microsoft/go-winio/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/Microsoft/go-winio
+
+go 1.12
+
+require (
+	github.com/pkg/errors v0.8.1
+	github.com/sirupsen/logrus v1.4.1
+	golang.org/x/sys v0.0.0-20190507160741-ecd444e8653b
```

Comment:
fwiw; double checked if this needed updating, but we're already ahead of this commit in master; https://github.com/golang/sys/compare/ecd444e8653b...4c4f7f33c9ed00de01c4c741d2177abfcfe19307